cbench/fakeswitch.c

```diff
@@ -376,20 +392,26 @@ void fakeswitch_handle_read(struct fakeswitch *fs)
             struct ofp_stats_request * stats_req;
             case OFPT_PACKET_OUT:
                 po = (struct ofp_packet_out *) ofph;
+                debug_msg(fs, "Got OFPT_PACKET_OUT, switch_status: %ld", fs->switch_status);
+                
                 if ( fs->switch_status == READY_TO_SEND && ! packet_out_is_lldp(po)) { 
-                    // assume this is in response to what we sent
+                    assume this is in response to what we sent
```

Comment:
Hey @zzxgzgz, will uncommenting line 398 fail the compile?

Reply:
agree, @zzxgzgz  please fix

Reply:
Fixed.